docs/controls/TabbedCommandBar.md

```diff
@@ -0,0 +1,248 @@
+
+```xaml
+<controls:TabbedCommandBar>
+  <controls:TabbedCommandBar.MenuItems>
+    <controls:TabbedCommandBarItem Header="Home">
+      <AppBarButton Icon="Undo" Label="Undo"/>
+      <AppBarButton Icon="Redo" Label="Redo"/>
+      <AppBarButton Icon="Paste" Label="Paste"/>
+    </controls:TabbedCommandBarItem>
+    <controls:TabbedCommandBarItem Header="View" IsContextual="True" Visibility="{Binding ElementName=ContextualToggle, Path=IsOn}">
+      <AppBarButton Label="Left" Icon="DockLeft"/>
+      <AppBarButton Label="Right" Icon="DockRight"/>
+      <AppBarButton Label="Bottom" Icon="DockBottom"/>
+    </controls:TabbedCommandBarItem>
+  </controls:TabbedCommandBar.MenuItems>
+</controls:TabbedCommandBar>
+
+...
+
+<ToggleSwitch x:Name="ContextualToggle" IsOn="True"
+              OffContent="Contextual Tab Off"
+              OnContent="Contextual Tab On"/>
+```
+
+`ComboBox`, `TextBox`, and `SplitButton` do not have "AppBar" variants. Below is how to use those controls in a `TabbedCommandBarItem`:
+
+```xaml
+<controls:TabbedCommandBar>
+  <controls:TabbedCommandBar.MenuItems>
+    <controls:TabbedCommandBarItem Header="Home">
+
+      <!-- SplitButton -->
+      <AppBarElementContainer>
+        <SplitButton>
+          <FontIcon Glyph="&#xE186;" FontSize="18" />
+          <SplitButton.Flyout>
+            <Flyout>
+              <controls:ColorPicker Margin="-10" Color="{ThemeResource Brand-Color}"/>
+            </Flyout>
+          </SplitButton.Flyout>
+        </SplitButton>
+      </AppBarElementContainer>
+
+      <!-- ComboBox -->
+      <AppBarElementContainer>
+        <ComboBox SelectedIndex="0" MinWidth="175">
+          <ComboBoxItem Content="Arial" />
+          <ComboBoxItem Content="Calibri" />
+          <ComboBoxItem Content="JetBrains Mono" />
+          <ComboBoxItem Content="Roboto" />
+          <ComboBoxItem Content="Sergio UI" />
+          <ComboBoxItem Content="Sergio UI Semibold" />
+        </ComboBox>
+      </AppBarElementContainer>
+
+      <!-- TextBox -->
+      <AppBarElementContainer>
+        <TextBox PlaceholderText="Size"/>
+      </AppBarElementContainer>
+      
+    </controls:TabbedCommandBarItem>
+  </controls:TabbedCommandBar.MenuItems>
+</controls:TabbedCommandBar>
+```
+
+## Sample Project
+
+[TabbedCommandBar Sample Page Source](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TabbedCommandBar). You can [see this in action](uwpct://Controls?sample=TabbedCommandBar) in the [Windows Community Toolkit Sample App](http://aka.ms/uwptoolkitapp).
+
+## Default Template
+
+[TabbedCommandBar XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBar.xaml) is the XAML template used in the toolkit for the default styling.
+
+## Requirements
+
+| Device family | Universal, 10.0.16299.0 or higher  |
```

Comment:
This needs to be 17763 (or the one up if you can't add conditional XAML to fix that other issue).

@kyaa-dost can you update the template.md doc for 17763 as our base? Also to remove the root url for any links going to docs.microsoft.com so they start with the '/' and whatever's above them as per #421 and #422. Feel free to remove the properties section as well for now based on our other discussions. Thanks!

Reply:
```suggestion
| Device family | Universal, 10.0.17763.0 or higher  |
```